### ISSUE TITLE
Drag even propagates to stage

### ISSUE THREAD

**Issue by vkaracic** [konvajs/vue-konva#71]
Hello!

I've just upgraded konva from 3.4.1 to 4.0.3. I have an issue with drag events propagating all the way to the stage.
Both the stage and every element I add to stage have `draggable: true` set, and, for example, if I add a rectangle to stage and attach a transformer to it, dragging any of the transformer anchors would transform the rectangle for a tiny amount and it would drag the stage in the direction that I drag my mouse.
Same thing happens if I try to drag elements on stage individually, on version 3.4.1 they would be dragged on the stage as you would expect but on 4.0.3 the entire stage is dragged with them resulting in the element being dragged by a very small amount and stage by full amount.

Do I need to add something like `@drag.prevent` on each element and transformer anchors or am I missing a part of setting the configuration for this to work properly?

The code is pretty big so I'll post an excerpt of relevant parts, let me know if you need more and I'll figure something out:
```
<div id="canvas">
    <v-stage ref="stage" :config="stageConfig" @click="stageClicked">
        <v-layer ref="layer">
              <v-text ref="text" :config="textConfig" />
        </v-layer>
    </v-stage>
</div>

stageConfig: {
  container: 'canvas',
  width: 3000,
  height: 800,
  draggable: true
}

textConfig: {
  x: 100,
  y: 100,
  draggable: true,
  offsetX: 0,
  offsetY: 0,
  rotation: 0,
  scaleX: 1,
  scaleY: 1,
  skewX: 0,
  skewY: 0,
  fontSize: 30,
  fill: '#000'
}

stageClicked(event) {
    if (event.target === this.stage) {
        this.deselectElement();
        this.transformer.detach();
    }
}
```

**Comment by lavrton** [comment]
Woh. That is a bad bug. Fixed with the `konva@4.0.4`. Please upgrade it and see how it works for you.

**Comment by vkaracic** [comment]
It works now! Thanks, @lavrton!
